scripts/config/org/icepear/echarts/origin/chart/treemap/treemap-series-breadcrumb-emphasis-item-style-option.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "BreadcrumbEmphasisItemStyleOption",
```

Comment:
The filename should keep consistent with the class/interface name. Please rename this file to `breadcrumb-emphasis-item-style-option.json`